.azuredevops/pipelineTemplates/jobs.validateModulePester.yml

```diff
@@ -172,7 +172,7 @@ jobs:
           testRunTitle: 'Global Module Tests'
           testResultsFormat: NUnit
           testResultsFiles: global-testResults.xml
-          failTaskOnFailedTests: false
+          failTaskOnFailedTests: true
```

Comment:
Is this from a different PR? what's the change?

Reply:
gone it is